elfi/loader.py

```diff
@@ -98,19 +98,19 @@ class RandomStateLoader(Loader):
     def load(cls, context, compiled_net, batch_index):
         key = 'output'
         seed = context.seed
-        if seed is False:
+        if seed is 'global':
```

Comment:
I think global is the term for the global numpy PRNG (and in this case it's not really a seed nor used but for debugging and testing).